qbft/commit.go

```diff
@@ -127,7 +135,7 @@ func validateCommit(
 	proposedMsg *SignedMessage,
 	operators []*types.Operator,
 ) error {
-	if err := baseCommitValidation(config, signedCommit, height, operators); err != nil {
+	if err := baseCommitValidation(config, signedCommit, height, operators, false); err != nil {
```

Comment:
if we go for the NoVerify suffix pattern, validateCommit could be validateCommitNoVerify to be explicit

Reply:
Agree